nat/service_ics.go

```diff
@@ -59,11 +58,17 @@ func (ics *serviceICS) Enable() error {
 }
 
 func (ics *serviceICS) enableRemoteAccessService() error {
-	status, err := ics.powerShell("Get-Service RemoteAccess | foreach { $_.StartType }")
+	status, err := ics.powerShell(`(Get-WmiObject Win32_Service -filter "Name='RemoteAccess'").StartMode`)
```

Comment:
Are we replacing because of `foreach` or because of `Get-Service`? Asking because we have `Set-Service` and `foreach` in other places too.

Reply:
Because of `Get-Service` here concretely. In 2.0 the Get-Service is not aware of the StartType property and does not have anything that's equivalent.